apps/client/src/features/operator/op-block/OpBlock.tsx

```diff
@@ -0,0 +1,13 @@
+import style from './OpBlock.module.scss';
+
+type Props = {
+  data: any;
```

Comment:
the type of this is OntimeBlock
https://github.com/cpvalente/ontime/blob/master/packages/types/src/definitions/core/OntimeEvent.type.ts#L22

Also, we have generally used a few contentions for props.
- props use `interface` while value types use `type`
- interfaces are usually named like `ComponentProps`, in this case it would be `OpBlockProps`

None of this affect the code in any relevant way, it is just a convention that the rest of the code follows and it is good not to diverge